portal/models/app_text.py

```diff
@@ -233,7 +234,7 @@ def name_key(**kwargs):
 
 class UnversionedResource(object):
     "Like VersionedResource for non versioned URLs (typically local)"
-    def __init__(self, url, asset=None):
+    def __init__(self, url, asset=None, variables={}):
```

Comment:
Classic python gotcha.  You don't want to assign a default variable to something like an empty dict.  Even if the current implementation doesn't modify `variables` - code changes over time and these problems sneak in.

I'd suggest making that `variables = None` and then in the body, assign variables to {} if needed.  Here's more on the topic:  http://effbot.org/zone/default-values.htm